CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Maintainers
+
+The Gaia repo is currently maintained by Aaron Blankstein (@kantai), and reviewers for PRs include
+@jcnelson and @zone117x
+
+# Submitting PRs
+
+A submitted PR must:
+
+1. Describe exactly what the goal of the PR is (and link to any relevant issues)
+2. Describe how that goal was achieved through the submitted implementation.
+3. The code must obey our eslint definitions, pass our unit tests, and
+   contain correct TypeFlow annotations.
+4. Contain tests that cover any and all new functionality or code changes.
+5. Describe how the new functionality can be tested manually.
+6. Document any new features or endpoints, and describe how developers
+   would be expected to interact with them.
+7. PR authors should agree to our contributor's agreement.
+
+PRs on Gaia should be reviewed by at least (2) maintainers.
```

Comment:
@kantai Do we want to set some squash and merge guidelines. Many projects require outside contributors to submit changes as a single commit (squash and merge). This allows for a quick rollback when problems arise.

Reply:
Yes -- though our preference is to do "non-squashing" merges. I'd rather see the whole history (apart from obvious commit spam).